jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -1008,6 +1014,22 @@ def triggerBWCTests(String buildManifestUrl, String platform, String distributio
     }
 }
 
+def triggerDistributionValidationWorkflow(String version) {
+    def triggerValidationWorkflow =
+            build job: 'distribution-validation',
+            propagate: true,
+            wait: true,
+            parameters: [
+                string(name: 'VERSION', value: version),
+                string(name: 'OS_BUILD_NUMBER', value: 'latest'),
```

Comment:
Do we need to consider when users don't select the current build as the `latest`? Can we set this build number to the current job number?

Reply:
But this is for OSD build right! current job number will be OSD_BUILD_NUMBER
I think we have to consider the case where we first trigger OS build, eventually wait for a while and build OSD . This way it will pick up the latest OS build while generating RC.

Reply:
Thats what we are doing right now. RC is not the first time we build a artifact for the given version. For validating OSD we just need some backend even if plugins are missing. Hence added `latest`. Also have not seen a scenario yet where we do not hit do not update latest